packages/neo-one-smart-contract-compiler/src/compile/syscalls.ts

```diff
@@ -985,7 +995,7 @@ export const SYSCALLS = {
   'Neo.Storage.Find': new SimpleSysCall({
     name: 'Neo.Storage.Find',
     args: [new SysCallArgument('context', StorageContextValue), new SysCallArgument('prefix', StorageKey)],
-    returnType: StorageIteratorValue,
+    returnType: StorageContextReadonlyValue,
```

Comment:
Why was this changed?

Reply:
I thought it was in a conversation... but can't find that -- so I'm reverting.